pkgs/http/lib/src/browser_client.dart

```diff
@@ -55,19 +57,22 @@ class BrowserClient extends BaseClient {
           'HTTP request failed. Client is already closed.', request.url);
     }
     var bytes = await request.finalize().toBytes();
-    var xhr = HttpRequest();
+    var xhr = XMLHttpRequest();
     _xhrs.add(xhr);
     xhr
-      ..open(request.method, '${request.url}', async: true)
```

Comment:
Why not keep this? \*shrug\*

Reply:
it *is* there. `async` is positional in the source API.